content/realtime/usage.textile

```diff
@@ -155,6 +156,18 @@ blang[objc,swift].
 
   **Note: Since version 1.0.8, the iOS client libraries are thread-safe.** Check "the README":https://github.com/ably/ably-cocoa#thread-safety for details.
 
+blang[flutter].
+  The Ably library is "hosted on Github":https://github.com/ably/ably-flutter and is available as a "Dart package":https://pub.dev/packages/ably-flutter/. Update your @pubspec.yaml@ with appropriate package version:
+
+  bc[yaml]. dependencies:
+    # ...
+    ably-flutter: [version]
+    # ...
+
+  Now, import the package in your dart file
```

Comment:
```suggestion
  Now, import the package into your Dart file.
```

I see there's another location in this PR where this will need fixing also.

Reply:
Addressed 4770624984d162f1ca4416b94db7dd50f643e400